core/trino-spi/src/test/java/io/trino/spi/type/TestingTypeManager.java

```diff
@@ -32,9 +35,39 @@
 public class TestingTypeManager
         implements TypeManager
 {
-    private static final List<Type> TYPES = ImmutableList.of(BOOLEAN, BIGINT, DOUBLE, INTEGER, VARCHAR, VARBINARY, TIMESTAMP_MILLIS, TIMESTAMP_WITH_TIME_ZONE, DATE, ID, HYPER_LOG_LOG);
-
-    private final TypeOperators typeOperators = new TypeOperators();
+    private static final TypeOperators typeOperators = new TypeOperators();
+    private static final List<Type> TYPES = ImmutableList.of(
+            BOOLEAN,
+            BIGINT,
+            DOUBLE,
+            INTEGER,
+            VARCHAR,
+            VARBINARY,
+            TIMESTAMP_MILLIS,
+            TIMESTAMP_WITH_TIME_ZONE,
+            DATE,
+            ID,
+            HYPER_LOG_LOG,
+            // types below are used in BenchmarkPartitionedOutputOperator
```

Comment:
`types below are used in BenchmarkPartitionedOutputOperator` ->
`types below are used in benchmarks (e.g BenchmarkPartitionedOutputOperator)`